### ISSUE TITLE
Test snapshot packages for v0.13.0

### ISSUE THREAD

**Issue by dcroote** [api3dao/airnode#1903]
Prior to the v0.13.0 release, the snapshot packages and corresponding Docker images need to be tested.

NPM packages:

```
@api3/airnode-abi@snapshot-v0.13.0
@api3/airnode-adapter@snapshot-v0.13.0
@api3/airnode-admin@snapshot-v0.13.0
@api3/airnode-deployer@snapshot-v0.13.0
@api3/airnode-examples@snapshot-v0.13.0
@api3/airnode-node@snapshot-v0.13.0
@api3/airnode-operation@snapshot-v0.13.0
@api3/airnode-protocol@snapshot-v0.13.0
@api3/airnode-utilities@snapshot-v0.13.0
@api3/airnode-validator@snapshot-v0.13.0
```

Docker images:

```
api3/airnode-deployer-dev:snapshot-v0.13.0
api3/airnode-client-dev:snapshot-v0.13.0
api3/airnode-admin-dev:snapshot-v0.13.0
```

Example of previous test suite: https://github.com/api3dao/airnode/issues/1839#issuecomment-1649162087

**Comment by dcroote** [comment]
tl;dr - everything worked

A checked box indicates success.

Test that npm packages admin, deployer, and validator, which have a --help CLI argument, are compiled correctly using npx

- [x] admin
  - [x] generate-mnemonic
- [x] deployer (tested more below)
- [x] validator
  - [x] Succeeds with --config "packages/airnode-validator/test/fixtures/config.valid.json" --secrets "packages/airnode-validator/test/fixtures/secrets.valid.env"

The following use the `coingecko-http-gateways` integration in airnode-examples on Ethereum goerli.

Test AWS deployment (took 98.79s for one deployment, 101.64s for another at a later time) with deployer Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
    - [x] **v0.13 specific** - the API response is present in `rawValue` independent of encoding success or failure
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] remove

Test GCP deployment (took 574.77s) with deployer Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] list
  - [x] info
  - [x] remove
